src/org/centos/pipeline/PackagePipelineUtils.groovy

```diff
@@ -117,21 +115,23 @@ def setDefaultEnvVars(Map envMap=null){
     // evaluating if env.ghprbActual is null, and if it's not, whether
     // it is something other than 'master'
     // If we're working with a staging instance:
-    //      We default to an MAIN_TOPIC of 'org.centos.stage'
+    //      We default to an MAIN_TOPIC of 'org.fedoraproject.stage'
     // If we're working with a production instance:
-    //      We default to an MAIN_TOPIC of 'org.centos.prod'
+    //      We default to an MAIN_TOPIC of 'org.fedoraproject.prod'
     // Regardless of whether we're working with staging or production,
     // if we're provided a value for MAIN_TOPIC in the build parameters:
 
     // We also set dataGrepperUrl which is needed for message tracking
     // and the correct jms-messaging message provider
 
     if (env.ghprbActualCommit != null && (env.ghprbActualCommit != "master" || env.ghprbPullId != "")) {
-        env.MAIN_TOPIC = env.MAIN_TOPIC ?: 'org.centos.stage'
+        env.MAIN_TOPIC_OLD = env.MAIN_TOPIC_OLD ?: 'org.centos.stage'
+        env.MAIN_TOPIC = env.MAIN_TOPIC ?: 'org.fedoraproject.stage'
         env.dataGrepperUrl = 'https://apps.stg.fedoraproject.org/datagrepper'
         env.MSG_PROVIDER = "fedora-fedmsg-stage"
     } else {
-        env.MAIN_TOPIC = env.MAIN_TOPIC ?: 'org.centos.prod'
+        env.MAIN_TOPIC_OLD = env.MAIN_TOPIC_OLD ?: 'org.centos.prod'
+        env.MAIN_TOPIC = env.MAIN_TOPIC ?: 'org.fedoraproject.prod'
```

Comment:
Last I knew, centos ci could not publish directly on org.fedoraproject.prod

Reply:
we have that define here:
https://pagure.io/fedora-ci/messages

will check with Fedora folks